### ISSUE TITLE
Allow Setting a timezone for ssm maintenance windows

### ISSUE THREAD

**Issue by twllight** [terraform-providers/terraform-provider-aws#6962]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
When Creating a ssm_maintenance_window, AWS allows for the addition of a timezone for that window as see here:
https://docs.aws.amazon.com/systems-manager/latest/userguide/reference-maintenance-windows-schedule-options.html
From what I can see within the code Terraform does not provide the ability to set a time zone for a maintenance window, which can be problematic when scheduling maintenance across a larger infrastructure. Providing a mechanism to add an optional timezone attribute to 
_aws_ssm_maintenance_window_ would be helpful.

### New or Affected Resource(s)

_aws_ssm_maintenance_window_

### Potential Terraform Configuration

```hcl
resource "aws_ssm_maintenance_window" "my_install_window" {
  name     = "My Maintenance Window"
  schedule = "cron(0 23 ? * *)"
  duration = "2"
  cutoff   = "0"
  timezone = "America/New_York"
}
```

### References
Here is the documentation regarding the timezone attribute within the ssm maintenance window for amazon:
https://docs.aws.amazon.com/systems-manager/latest/userguide/reference-cron-and-rate-expressions.html
As well as the go sdk regarding the same:
https://docs.aws.amazon.com/sdk-for-go/api/service/ssm/#SSM.CreateMaintenanceWindow

Thank you!

**Comment by bflad** [comment]
Pull request submitted to add `end_date`, `schedule_timezone`, and `start_date` attributes: #7040

**Comment by bflad** [comment]
Support for the new attributes mentioned above has been merged and will release with version 1.55.0 of the Terraform AWS provider, likely middle of this week.

**Comment by bflad** [comment]
This has been released in [version 1.55.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1550-january-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
